osidb/models.py

```diff
@@ -1794,6 +1794,34 @@ def fix_state(self):
     def is_closed(self):
         return self.status.upper() == "CLOSED"
 
+    @property
+    def is_acked(self):
+        """
+        tracker acked by ProdSec - filed against an acked stream
+        the default approach important and critical aggregated impact
+        """
+        return not self.is_unacked
+
+    @property
+    def is_unacked(self):
+        """
+        tracker not acked by ProdSec - filed against an unacked stream
+        the default approach low and moderate aggregated impact
+        """
+        # in the case of invalid trackers let us default to acked
+        if not PsUpdateStream.objects.filter(name=self.ps_update_stream).exists():
+            return False
+
+        unacked_ps_update_stream = (
+            PsUpdateStream.objects.filter(name=self.ps_update_stream)
+            .get()
+            .ps_module.unacked_ps_update_stream.first()
+        )
+        if unacked_ps_update_stream is None:
+            return False
+
+        return self.ps_update_stream == unacked_ps_update_stream.name
```

Comment:
I wonder whether this could be simplified to something like this:

```python
unacked_ps_update_stream = PsUpdateStream.objects.filter(name=self.ps_update_stream, unacked_to_ps_module__isnull=True).first()

if unacked_ps_update_stream is None:
    return False
return True

```

Reply:
Great hint! Actually it should be `isnull=False` :smile: but otherwise with this the method can be shortened to a single expression.

Reply:
Ah, I always confuse this expression, `isnull=True` somehow translates in my head to just `True` :grin: